packages/cf.js/src/utils/uuid.ts

```diff
@@ -0,0 +1,60 @@
+/* tslint:disable */
```

Comment:
I think this whole file can be linted. e.g., `var` vs `const` and typings etc.

Reply:
I had issues with linting that I didn't really want to start getting into. This is highly tested JS code :/ If its important I can look into it tomorrow

Reply:
For example this: `[i++]` throws `[tslint] Forbidden ++ operator [no-increment-decrement]`

Reply:
Ah.

Yeah maybe we should make it a `.js` file then and import it that way?

Reply:
Removed this file so resolving